backend/uclapi/oauth/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from .app_helpers import generate_user_token
+
+
+class OAuthScope(models.Model):
+    # We should really have a primary key for the OneToOneField
+    id = models.AutoField(primary_key=True)
```

Comment:
this is added by Django implicitly, just FYI 😊

Reply:
yeah I know this now